app/models/compute_resources/foreman/model/vmware.rb

```diff
@@ -320,7 +320,7 @@ def create_vm(args = { })
 
       args = parse_networks(args)
       args = args.with_indifferent_access
-      if args[:image_id].present?
+      if args[:provision_method] == 'image'
```

Comment:
I'm not sure if this gets passed through in the compute orchestration (without having tested it).  The provision_method is an attribute of the host, not part of the compute attributes.

Only the compute attributes are passed into the CR's create_vm method over here: https://github.com/theforeman/foreman/blob/1.10.0-RC3/app/models/concerns/orchestration/compute.rb#L76

We might need to merge the provision_method from the host into that hash to make it accessible.